### ISSUE TITLE
Ingest sst file does not overwrite the old data

### ISSUE THREAD

**Issue by Nicole00** [vesoft-inc/nebula#2722]
After the SST file was imported through INGEST, it doesn't overwrite the data that was previously inserted.

operations：
1. Insert a point data: insert vertex user(name) values ​​1:("tom")
2. Generate sst based on csv file, csv has only two records of data: (the first col is vid, the second col is name)
```  
1, jina
2, tina
```
3. After INGEST is imported, the `show stats` shows two records, and the vertex with id 2 can be found through the fetch statement. The name of the vertex with id 1 is still `tom`, which has not been overwritten with `jina`.

extra info：
version： 2.5.0 rpm
![image](https://user-images.githubusercontent.com/16240361/131092194-83d8baba-c883-4f06-a20d-900e2238e186.png)

![image](https://user-images.githubusercontent.com/16240361/131091496-bc288a06-c7d9-4946-8689-00cbec14d89b.png)

**Comment by darionyaphet** [comment]
I have try to replay this case, but it didn't happen.

First, insert some vertex into the database and then ingest an SST file. I can read the new value created by ingest file.

**Comment by darionyaphet** [comment]
```
(root@nebula) [test]> CREATE TAG t(name string)
Execution succeeded (time spent 2765/3299 us)

Sun, 19 Sep 2021 05:39:59 CST

(root@nebula) [test]> INGEST
Execution succeeded (time spent 297210/297759 us)

Sun, 19 Sep 2021 05:40:50 CST

(root@nebula) [test]> FETCH PROP ON t 1
+------------------------+
| vertices_              |
+------------------------+
| (1 :t{name: "darion"}) |
+------------------------+
Got 1 rows (time spent 4338/5118 us)

Sun, 19 Sep 2021 05:40:54 CST

(root@nebula) [test]> FETCH PROP ON t 2
+------------------------+
| vertices_              |
+------------------------+
| (2 :t{name: "yaphet"}) |
+------------------------+
Got 1 rows (time spent 3404/4088 us)

Sun, 19 Sep 2021 05:40:56 CST

(root@nebula) [test]> INGEST
Execution succeeded (time spent 289964/290602 us)

Sun, 19 Sep 2021 05:41:33 CST

(root@nebula) [test]> FETCH PROP ON t 1
+-------------------------------+
| vertices_                     |
+-------------------------------+
| (1 :t{name: "darion.yaphet"}) |
+-------------------------------+
Got 1 rows (time spent 4251/5015 us)

Sun, 19 Sep 2021 05:41:36 CST

(root@nebula) [test]> INSERT VERTEX t(name) VALUES 1:("d")
Execution succeeded (time spent 7870/8561 us)

Sun, 19 Sep 2021 05:43:51 CST

(root@nebula) [test]> FETCH PROP ON t 1
+-------------------+
| vertices_         |
+-------------------+
| (1 :t{name: "d"}) |
+-------------------+
Got 1 rows (time spent 3697/4386 us)

Sun, 19 Sep 2021 05:43:53 CST

(root@nebula) [test]> INGEST
Execution succeeded (time spent 291909/292424 us)

Sun, 19 Sep 2021 05:43:56 CST

(root@nebula) [test]> FETCH PROP ON t 1
+-------------------------------+
| vertices_                     |
+-------------------------------+
| (1 :t{name: "darion.yaphet"}) |
+-------------------------------+
Got 1 rows (time spent 4246/5040 us)

Sun, 19 Sep 2021 05:43:59 CST
```

**Comment by darionyaphet** [comment]
you should set enable_vertex_cache is false to make sure ingest SST file into system

**Comment by wey-gu** [comment]
> you should set enable_vertex_cache is false to make sure ingest SST file into the system

I think this should be documented/highlighted in SST chapters

cc @randomJoe211 @cooper-lzy

**Comment by cooper-lzy** [comment]
> > you should set enable_vertex_cache is false to make sure ingest SST file into the system
> 
> I think this should be documented/highlighted in SST chapters
> 
> cc @randomJoe211 @cooper-lzy

It is only updated in 2.5.0 and 2.5.1 branch, because the parameter enable_vertex_cache has not been available since 2.6.

![image](https://user-images.githubusercontent.com/78672629/134485753-5de2cb76-f71b-4819-8e7f-7cfcde490c7d.png)

**Comment by wey-gu** [comment]
> > > you should set enable_vertex_cache is false to make sure ingest SST file into the system
> > 
> > 
> > I think this should be documented/highlighted in SST chapters
> > cc @randomJoe211 @cooper-lzy
> 
> It is only updated in 2.5.0 and 2.5.1 branch, because the parameter enable_vertex_cache has not been available since 2.6.
> 
> ![image](https://user-images.githubusercontent.com/78672629/134485753-5de2cb76-f71b-4819-8e7f-7cfcde490c7d.png)

Sorry, I didn't realize it's already there, thanks!!!
